target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -71,15 +84,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:lan:1" "3:lan:4" "4:lan:3" "5:lan:2" "2:wan"
 		;;
-	comfast,cf-e5|\
+	comfast,cf-e110n-v2|\
+	comfast,cf-e120a-v3|\
 	glinet,gl-ar150|\
-	glinet,gl-ar300m-nand|\
-	glinet,gl-ar300m-nor|\
-	glinet,gl-x750|\
-	tplink,tl-wr810n-v1|\
-	tplink,tl-wr810n-v2|\
-	ubnt,routerstation|\
-	yuncore,a770)
```

Comment:
This whole removal block is not recreated, except ubnt,routerstation below. (rebase issue?)

Reply:
All of these uses the last default case.